### ISSUE TITLE
Sharing a File error because of moved directory

### ISSUE THREAD

**Issue by stefan-franz** [nextcloud/android#11421]
### ⚠️ Before posting ⚠️

- [X] This is a **bug**, not a question or an enhancement.
- [X] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [X] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

I moved a directory of my Nextcloud. Now if I I want on Android share a file over the nextcloud Android App I get an error, because the nextcloud Android App stands still in that directory that doesn't exist on this place. 

### Cause of error
```java
Exception in thread "Thread-17" java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String com.owncloud.android.datamodel.OCFile.getRemotePath()' on a null object reference
    at com.owncloud.android.operations.RefreshFolderOperation.run(RefreshFolderOperation.java:230)
    at com.owncloud.android.lib.common.operations.RemoteOperation.run(RemoteOperation.java:399)
    at java.lang.Thread.run(Thread.java:923)

```

### App information
* ID: `com.nextcloud.client`
* Version: `30240190`
* Build flavor: `gplay`

### Device information
* Brand: `motorola`
* Device: `nairo`
* Model: `moto g 5G plus`
* Id: `RPNS31.Q4U-39-27-9-2-9`
* Product: `nairo_retaile`

### Firmware
* SDK: `30`
* Release: `11`
* Incremental: `9ab668`

### Expected behaviour

A possibility to move to the root directory whoild great. 

### Actual behaviour

An error message as poster of I try to move to the root directory 

### Android version

Current version 

### Device brand and model

Moto g 5G

### Stock or custom OS?

Stock

### Nextcloud android app version

3.24.1

### Nextcloud server version

25.0.4

### Using a reverse proxy?

No

**Comment by xolyu** [comment]
I had encountered exactly the same problem.

My 1st attempt was unfortunately unsuccessful: to create a folder with that name again.

My workaround to continue using the app and uploading data:

I have configured accounts from two Nextcloud instances in the app. When switching from one account to the other, the app switches to the root directory. By switching to account B and back to account A, I was then able to navigate through the directories again.
